diff --git a/chrome/BUILD.gn b/chrome/BUILD.gn
--- a/chrome/BUILD.gn
+++ b/chrome/BUILD.gn
@@ -1319,6 +1319,7 @@
       "//chrome/browser:startup_data",
       "//chrome/browser/lifetime",
       "//chrome/browser/mac",
+      "//chrome/browser/netnyahoo:exports",  # Netnyahoo: engine/chromium
       "//chrome/common:buildflags",
       "//chrome/common:version_header",
       "//chrome/common/profiler",
